indigo_app/templates/indigo_api/_task_detail.html

```diff
@@ -116,6 +116,31 @@ <h5>
 
                 <div class="text-right">
                   <input class="btn btn-primary" type="submit" value="Add comment" id="id_submit" disabled>
+                  
+                  {% if task.state == 'pending_review' %}
+                    {% if task.unsubmit_task_permission %}
+                      <input
+                        class="btn btn-primary mr-2 float-left" 
+                        type="submit" 
+                        data-confirm="Are you sure you want to request changes to this task?" 
+                        formaction="{% url 'unsubmit_task' place=place.place_code pk=task.pk %}" 
+                        id="comment_request_changes"
```

Comment:
Pedantic, but i don't think you need to explicitly include that this is a comment button -- "request_changes" is sufficient I think. The fact that there could be a comment linked to the button is incidental and doesn't impact the identity of the button.

Reply:
I'll apply the same to the 'approve' button